prisma/schema.prisma

```diff
@@ -1,5 +1,6 @@
 generator client {
-  provider = "prisma-client-js"
+  provider      = "prisma-client-js"
+  binaryTargets = ["native", "linux-musl-arm64-openssl-3.0.x"]
```

Comment:
Could u elaborate on why this is needed?

Reply:
This is because I was facing such error on m1 mac when running the container with compose.
https://stackoverflow.com/questions/76144590/prismaclientinitializationerror-query-engine-library-for-current-platform-linu

I guess it maybe due to not able to identify the correct engine for that platform and I explicitly set it to that value